modules/tabs-v2/assets/scss/frontend.scss

```
@@ -2,128 +2,127 @@
 // Tabs v2
 //
 .elementor-widget-tabs-v2 {
-	--tabs-v2-wrapper-display: flex;
-	--tabs-v2-wrapper-justify-content: start;
-	--tabs-v2-wrapper-title-size: auto;
+	--tabs-v2-default-padding: 15px;
+
+	--tabs-v2-display: flex;
+	--tabs-v2-direction: column;
+	--tabs-v2-gap: 10px;
+
+	--tabs-v2-tabs-wrapper-display: flex;
+	--tabs-v2-tabs-wrapper-direction: row;
+	--tabs-v2-tabs-wrapper-grow: initial;
+	--tabs-v2-tabs-wrapper-justify-content: initial;
+	--tabs-v2-tabs-wrapper-width: initial;
+
+	--tabs-v2-height: initial;
+	--tabs-v2-overflow-x: hidden;
+	--tabs-v2-overflow-y: auto;
+
 	--tabs-v2-border-width: 1px;
 	--tabs-v2-border-color: #{$gray-light};
 	--tabs-v2-background-color: transparent;
-	--tabs-v2-vertical-navigation-width: 25%;
 
-	/* TODO Find better solution for all global defaults. */
-	--tabs-v2-title-color: var(--e-global-color-primary);
-	--tabs-v2-title-active-color: var(--e-global-color-accent);
+	--tabs-v2-title-color: var( --e-global-color-secondary, #54595F );
```

Use a variable for the fallback.

```suggestion
	--tabs-v2-title-color: var( --e-global-color-secondary, #54595F );
```

---

updated